app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):

```

Comment:
No need to inherit additionally from ABC. If the class does not implement abstact methods - it will remain abstract